bin/inbox-start.py

```diff
@@ -142,8 +142,8 @@ def main(prod, enable_tracer, enable_profiler, config, process_num, exit_after):
 
     sync_service = SyncService(process_identifier, process_num)
 
-    signal.signal(signal.SIGTERM, sync_service.stop)
-    signal.signal(signal.SIGINT, sync_service.stop)
+    signal.signal(signal.SIGTERM, lambda *_: sync_service.stop())
+    signal.signal(signal.SIGINT, lambda *_: sync_service.stop())
```

Comment:
I made this mistake too many times b/c too much branches. `stop()` now does not accept arguments and signal handlers do.